lib/graphql/static_validation/validator.rb

```diff
@@ -36,6 +36,7 @@ def validate(query, validate: true)
             rules_to_use.each do |rule_class_or_module|
               if rule_class_or_module.method_defined?(:validate)
                 rule_class_or_module.new.validate(context)
+                break if context.errors.size >= @schema.max_validation_errors
```

Comment:
I _think_ calling `.validate(context)` _attaches_ some visitor hooks, but the hooks themselves don't run until `.visit` below. Are there any contexts where this condition is triggered?

Reply:
I dug into this a bit and I think this code might actually just be dead? As far as I can tell, none of the `GraphQL::StaticValidation::ALL_RULES` actually define a `validate` method at all anymore?